test/integration/targets/aws_s3/tasks/main.yml

```diff
@@ -252,3 +252,76 @@
     that:
       - result.changed == False
 # ============================================================
+- name: create a tempfile for multipart test
+  tempfile:
+  register: tmp1
+
+- name: make tempfile 4 GB
+  command:
+    _raw_params: "dd if=/dev/zero of={{ tmp1.path }} bs=1m count=4096"
```

Comment:
Works on mac os x. I need to fix this.

Reply:
This also works on linux if you capitalize the M in `bs=1M`

Reply:
But then it doesn't work on os x...
```
$ dd if=/dev/zero of=/tmp/test.txt bs=1M count=4096
dd: bs: illegal numeric value
```